eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -87,6 +87,10 @@ jobs:
     - ${{ if eq(parameters.platform, 'Linux_s390x') }}:
         - Ubuntu.2004.S390X.Experimental.Open
 
+    # Linux PPC64le
+    - ${{ if eq(parameters.platform, 'Linux_ppc64le') }}:
+        - Ubuntu.2004.PPC64EL.Experimental.Open
```

Comment:
> Ubuntu.2004.PPC64EL.Experimental.Open

Is this a typo, or should it really be "EL"?

Reply:
Yes? No? Maybe? The PR I filed asked for `le` - https://github.com/dotnet/arcade/issues/9567 but the internal PR adding it used `el` - https://dev.azure.com/dnceng/internal/_git/dotnet-helix-machines/pullrequest/23425 and Debian uses `el` as a joke ("Endian Little" instead of "Little Endian" as a byte ordering joke) so...

Either way, I can't get Helix to acknowledge the queue's existence so I'm waiting for feedback from core-eng either way.